### ISSUE TITLE
MTEX 4.x

### ISSUE THREAD

**Issue by ralfHielscher** [edwinofsakh/ebsdam#29]
Hi Edwin,

do you have any plans to make this working with MTEX 4.x?

Ralf.

**Comment by edwinofsakh** [comment]
Yes, I have. At first I plan go to MTEX 3.5, but after this go to MTEX 4.x.
